corrector.rb

```diff
@@ -0,0 +1,5 @@
+class Corrector
+  def correct_name(name)
+    name.capitalize.slice(0, 10)
+  end
+end
```

Comment:
- [ ] Here you should have an Interface (a Namable class) with a method 'correct_name' **rasing** an error. Let's fix that